plugins/nvm/nvm.plugin.zsh

```diff
@@ -1,3 +1,4 @@
 # The addition 'nvm install' attempts in ~/.profile
 
 [[ -s ~/.nvm/nvm.sh ]] && . ~/.nvm/nvm.sh
+[[ -r $NVM_DIR/bash_completion ]] && . $NVM_DIR/bash_completion
```

Comment:
NVM does [load `bashcompinit` itself, as part of loading that bash completion file](https://github.com/creationix/nvm/blob/1e2a7a8de447c9b872fbf4cbe6c959cfc8d02a55/bash_completion#L84-L89) (but not as part of `nvm.sh`, I think).

But you should not need to use it, because `plugins/nvm` provides a zsh-specific `_nvm` completion function. The bash completion file's location under `$NVM_DIR` may change depending on the install method; e.g. Homebrew's formula relocates it to `$(brew --prefix nvm)/etc/bash_completion.d/nvm`.